src/Microsoft.AspNet.Mvc.TagHelpers/LinkTagHelper.cs

```diff
@@ -355,13 +356,12 @@ private void BuildFallbackBlock(TagHelperContent builder)
                 // indicating that the primary stylesheet failed to load.
                 builder
                     .AppendHtml("<script>")
-                    .AppendHtml(
-                        string.Format(
-                            CultureInfo.InvariantCulture,
-                            JavaScriptResources.GetEmbeddedJavaScript(FallbackJavaScriptResourceName),
-                            JavaScriptEncoder.Encode(FallbackTestProperty),
-                            JavaScriptEncoder.Encode(FallbackTestValue),
-                            JavaScriptStringArrayEncoder.Encode(JavaScriptEncoder, fallbackHrefs)))
+                    .AppendFormat(
+                        CultureInfo.InvariantCulture,
+                        JavaScriptResources.GetEmbeddedJavaScript(FallbackJavaScriptResourceName),
+                        new HtmlString(JavaScriptEncoder.Encode(FallbackTestProperty)),
+                        new HtmlString(JavaScriptEncoder.Encode(FallbackTestValue)),
+                        new HtmlString(JavaScriptStringArrayEncoder.Encode(JavaScriptEncoder, fallbackHrefs)))
```

Comment:
This overall change is basically a wash on @rynowak's BasicViews test site with `asp-fallback-*` attributes added to `<link>` and `<script>` elements. However this simplification alone is negative. Higher numbers in the comparison below all tie directly to the following block of code. I'm leaning toward checking this in and then fixing up the new HtmlAbstractions repo, special-casing `HtmlEncodedString` there and changing `HtmlString` to inherit from `HtmlEncodedString` here. Any objections?
``` c#
public string Format(string format, object arg, IFormatProvider formatProvider)
{
    // This is the case we need to special case. We trust the IHtmlContent instance to do the
    // right thing with encoding.
    var htmlContent = arg as IHtmlContent;
    if (htmlContent != null)
    {
        using (var writer = new StringWriter())
        {
            htmlContent.WriteTo(writer, _encoder);
            return writer.ToString();
        }
    }
    ...
}
```

![tag 0](https://cloud.githubusercontent.com/assets/6431421/11522826/c8f2a428-9870-11e5-942a-ca7386d55f1d.png)
![tag 1](https://cloud.githubusercontent.com/assets/6431421/11522828/ce95bbc2-9870-11e5-8275-2db266d8fce2.png)